### PR TITLE
update travis ci conf

### PR DESCRIPTION
**Pull request by andyxning**
This PR depends on #81 which should be merged first.

* Update travis ci configuration using Makefile
* adding test for golang 1.7 :)

/cc @brancz @vdavidoff

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kube-state-metrics/82)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 language: go
 go:
   - 1.6
```
Comment (piosz) [inline .travis.yml]: do we need to test against go 1.6?

**Comment by piosz** [comment]
LGTM

**Comment by brancz** [comment]
can you rebase and remove testing against 1.6? then lgtm

**Comment by andyxning** [comment]
@brancz Will do.

File: .travis.yml (hunk 1 above)
Comment (andyxning) [inline .travis.yml]: @piosz Just want to add go 1.7 since other kubernetes repo seems to migrate to it. I have no favor on testing agaginst go 1.6. :)

**Comment by andyxning** [comment]
@brancz @piosz  Comments are addressed. PTAL.

**Comment by brancz** [comment]
lgtm

**Pull request merged**